x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -118,34 +118,21 @@ export interface PolicyDetailsState {
  * Endpoint Policy configuration
  */
 export interface PolicyConfig {
-  windows: {
-    events: {
-      process: boolean;
-      network: boolean;
-    };
-    /** malware mode can be off, detect, prevent or prevent and notify user */
-    malware: MalwareFields;
+  windows: UIPolicyConfig['windows'] & {
```

Comment:
curious to know why this was needed

Reply:
I was trying random stuff. Don't think the change is needed